### PR TITLE
Attempt to fix windows build.

### PR DESCRIPTION
**Pull request by mlyle**

### CODE REVIEW

**Comment by mlyle** [comment]
jenkins, build artifacts please

**Comment by mlyle** [comment]
jenkins, build artifacts please

File: flight/PiOS/posix/pios_sys.c [hunk 1]
```diff
@@ -639,13 +639,15 @@ void PIOS_SYS_Args(int argc, char *argv[]) {
 				hw_argseen = false;
 				break;
 			}
+#if !(defined(_WIN32) || defined(WIN32) || defined(__MINGW32__))
```
Comment (tracernz) [inline flight/PiOS/posix/pios_sys.c:648]: Need to do the same in usage.

Comment (mlyle) [reply flight/PiOS/posix/pios_sys.c:648]: Right now I don't do *anything* to make usage correct / limit to what's applicable :/

Comment (mlyle) [reply flight/PiOS/posix/pios_sys.c:648]: Oh I guess I do a little.  Will look.  Usually not worth blocking ab uild fix for something like this

Comment (mlyle) [reply flight/PiOS/posix/pios_sys.c:648]: OK, it's done, but not really correct still (top line of summary doesn't change).

**Review by tracernz — APPROVED** [review]

**Pull request merged**